app/models/miq_product_feature.rb

```diff
@@ -153,6 +153,58 @@ def self.seed_feature(hash, features)
     return feature, status
   end
 
+  def self.find_all_by_identifier(features)
+    where(:identifier => features)
+  end
+
+  def self.obj_features
+    @obj_cache ||= begin
+      # create hash with parent identifier and details
+      features = select('*').select(arel_attribute(:parent_identifier).as("parent_identifier")).each_with_object({}) do |f, h|
```

Comment:
Think `select` api has been enhanced since this was written. Could you test with:

```ruby
select('*').select(:parent_identifier).each_with_object({})
```